lib/api/controllers/server.js

```diff
@@ -89,67 +89,76 @@ class ServerController extends NativeController {
     // and default rights
     delete config.security;
 
-    return Bluebird.resolve(config);
+    if (config.VAULT_KEY) {
+      delete config.VAULT_KEY
+    }
+
+    return config
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)